src/Service/Context/PageContext.php

```diff
@@ -293,23 +335,37 @@ private function getMetaTagRenderArray($name, $content) {
   }
 
   /**
-   * Set page context title.
+   * Get the render array for a JavaScript tag.
+   *
+   * @return array
+   *   The render array
+   */
+  private function getJavaScriptTagRenderArray() {
+    return [
+      [
+        '#tag' => 'script',
+        '#attributes' => [
+          'src' => $this->jsPath,
+        ],
+      ],
+      'acquia_lift_javascript',
+    ];
+  }
+
+  /**
+   * Populate page's HTML head.
    *
-   * @param array|string $title
-   *   Set page context title.
+   * @param &$htmlHead
+   *   The HTML head that is to be populated.
    */
-  public function setPageContextTitle($title) {
-    // If markup, strip tags and convert to string.
-    if (is_array($title) && isset($title['#markup']) && isset($title['#allowed_tags'])) {
-      $allowed_tags = empty($title['#allowed_tags']) ? '' : '<' . implode('><', $title['#allowed_tags']) . '>';
-      $title = strip_tags($title['#markup'], $allowed_tags);
-    }
-    // If still an array or empty, set title to empty.
-    if (is_array($title) || empty($title)) {
-      $this->pageContext['content_title'] = '';
-      return;
+  public function populateHtmlHead(&$htmlHead) {
+    // Attach Lift's metatags.
+    foreach ($this->pageContext as $name => $content) {
+      $renderArray = $this->getMetaTagRenderArray($name, $content);
+      $htmlHead[] = [$renderArray, $name];
```

Comment:
Well, that's exactly the comment I would add there. But I'll leave it to you :-)